src/main/cpp/src/CastStringJni.cpp

```diff
@@ -196,7 +196,7 @@ JNIEXPORT jlong JNICALL Java_com_nvidia_spark_rapids_jni_CastStrings_toIntegersW
 }
 
 JNIEXPORT jlong JNICALL Java_com_nvidia_spark_rapids_jni_CastStrings_fromIntegersWithBase(
-  JNIEnv* env, jclass, jlong input_column, jint base)
+  JNIEnv* env, jclass, jlong input_column, jint base, jboolean drop_leading_zeros)
```

Comment:
It looks odd that the new parameter applies only to base=16 but we pass it in the general API for all bases. I would make an extra method just for wiring a raw strings::integers_to_hex(input_view) to JNI

Reply:
That API should probably live in cudf repo if it's just a wrapper around `integers_to_hex`.